app/controllers/event.js

```diff
@@ -33,6 +34,15 @@ export default Ember.Controller.extend({
     return Ember.copy(this.get('queryParamsHash'));
   },
 
+  //These centralized options are stored in the discover root controller
+  aggOptions: Ember.computed('discoverController', function() {
```

Comment:
I like your instinct to keep that information in one place, but is there a better way to share that information without one controller needing to ask another?